libpod/oci.go

```diff
@@ -318,6 +318,7 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string) (er
 	// 0, 1 and 2 are stdin, stdout and stderr
 	cmd.Env = append(r.conmonEnv, fmt.Sprintf("_OCI_SYNCPIPE=%d", 3))
 	cmd.Env = append(cmd.Env, fmt.Sprintf("_OCI_STARTPIPE=%d", 4))
+	cmd.Env = append(cmd.Env, fmt.Sprintf("XDG_RUNTIME_DIR=%s", os.Getenv("XDG_RUNTIME_DIR")))
```

Comment:
Is this a runc-ism?

Reply:
yes, it is used by runc to find the path where to write the status for the rootless containers.

Reply:
You can manually override with `--root`. `XDG_RUNTIME_DIR` is just used as the default (instead of `/run` which is the default when running as `root`).